### PR TITLE
Spring boot and byte buddy status update

### PR DESCRIPTION
**Pull request by SaaiVenkat**
- The mentioned two tests were moved to Gradle. I have fixed them, but I haven't raised a PR yet.
  - `org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.requestMappingIsInstrumented`
  - `org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented`
- The developer of byte-buddy has reconsidered and fixed one test based on my fix.

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -4677,8 +4677,8 @@ https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd14
...
-https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented,NIO,,,https://github.com/TestingResearchIllinois/idoft/issues/411
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.requestMappingIsInstrumented,NIO,MovedToGradle,https://github.com/spring-projects/spring-boot/commit/ce99db19028b54886ae99c1fd916e613a982b65f,https://github.com/TestingResearchIllinois/idoft/issues/411
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented,NIO,MovedToGradle,https://github.com/spring-projects/spring-boot/commit/ce99db19028b54886ae99c1fd916e613a982b65f,https://github.com/TestingResearchIllinois/idoft/issues/411
```
Comment (darko-marinov) [inline pr-data.csv:4681]: Move the commit URL from `PR Link` (second-last column) to `Notes` (last column). In `Notes` use `;` to separate two links.

File: pr-data.csv [hunk 2]
```diff
@@ -4464,7 +4464,7 @@ https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,qu
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.ordering,NIO;NOD,Accepted,https://github.com/querydsl/querydsl/pull/2658,
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.paging,NOD,Accepted,https://github.com/querydsl/querydsl/pull/2658,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/45
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.singleResult,OD-Brit,,,
-https://github.com/raphw/byte-buddy,b19eabacf6a9df26641052037666566b2152ce9f,byte-buddy-dep,net.bytebuddy.ClassFileVersionTest.testLatestVersion,ID,DeveloperWontFix,,
+https://github.com/raphw/byte-buddy,b19eabacf6a9df26641052037666566b2152ce9f,byte-buddy-dep,net.bytebuddy.ClassFileVersionTest.testLatestVersion,ID,InspiredAFix,https://github.com/raphw/byte-buddy/pull/1547,
```
Comment (darko-marinov) [inline pr-data.csv:4467]: Why not `Accepted` here? The developer merged your PR.

File: gr-data.csv [hunk 3]
```diff
@@ -473,6 +473,8 @@ https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsList,ID-HtF,,,
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsRegEx,ID-HtF,,,
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsUpdate,ID-HtF,,,
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.requestMappingIsInstrumented,NIO,,,
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented,NIO,,,
```
Comment (darko-marinov) [inline gr-data.csv:477]: Are these `NIO` when run in the same JVM or even in different JVMs?

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Please address several comments as soon as possible if you want me to accept by Thursday.

File: gr-data.csv (hunk 3 above)
Comment (SaaiVenkat) [reply gr-data.csv:477]: I have tested them in the same JVM and they are `NIO`.

File: pr-data.csv (hunk 2 above)
Comment (SaaiVenkat) [reply pr-data.csv:4467]: The developer added his own code based on mine. So, this will be a InspiredAFix, but others will be Accepted

**Comment by SaaiVenkat** [comment]
> Please address several comments as soon as possible if you want me to accept by Thursday.

I have addressed your comments Professor.

**Pull request merged**